mpc_follower_wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,216 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mpc_follower_wrapper)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+add_compile_options(-std=c++11)
+set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+add_definitions(-DBOOST_LOG_DYN_LINK=1)
+set(CMAKE_BUILD_TYPE Debug)
+set(CMAKE_CXX_FLAGS "-lboost_system") 
```

Comment:
No sure why these lines are nessasary
```
add_definitions(-DBOOST_LOG_DYN_LINK=1)
set(CMAKE_BUILD_TYPE Debug)
set(CMAKE_CXX_FLAGS "-lboost_system") 
```

None of our other boost packages use them